crates/proxy-server/Cargo.toml

```diff
@@ -13,4 +13,6 @@ libc = "0.2"
 log = "0.4"
 socksv5 = {version="0.3", default-features=false, features=["tokio"]}
 tokio = {workspace=true, features=["full", "tracing"]}
+
+[target.'cfg(target_os = "linux")'.dependencies]
```

Comment:
```rust
error[E0425]: cannot find function `splice` in crate `libc`
  --> /Users/djc/.cargo/registry/src/index.crates.io-6f17d22bba15001f/tokio-splice-0.2.1/src/io.rs:40:15
   |
40 |         libc::splice(
   |               ^^^^^^ not found in `libc`
```

Perhaps these should be fixed in tokio-splice/libc, but since this crate already only calls the function on Linux, this seems actually correct.